data/internship.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+
+export default {
+    items: [
+        {
+            date: '9/27',
+            name: 'Fall General Meeting',
+            info: 'Come learn about the different ACM committees!',
+            active: true,
+        },
+        {
+            date: '10/10',
+            name: 'Application Form is Open',
+            info: 'Fill out the application form for up to 3 committees that you would like to intern for',
+            active: true,
+        },
+        {
+            date: '10/22',
+            name: 'Applications Due',
+            info: 'Submit your application by 11:59 pm',
+            active: false,
+        },
+        {
+            date: '10/22-11/02',
+            name: 'Interviews',
+            info: 'A handful of applicants will be chosen for an interview with the committee(s) you applied to',
+            active: false,
+        },
+        {
+            date: '11/03-11/04',
+            name: 'Decisions Released',
+            info: 'Hear back via Email what committee(s) accepted you (if you applied to multiple, decide which one you would like to intern for)',
+            active: false,
+        },
+        {
+            date: '11/06',
+            name: 'Official ACM Onboarding',
+            info: 'Learn what you need to know as an ACM Intern',
+            active: false,
+        },
+    ],
+    testimonials: [
+        {
+            name: 'Maya Raman',
+            quote: <>
+                <b>What is your favorite memory from being an intern?</b><br/>
+                Accidentally breaking tables at the intern retreat
+                <br/><br/><b>How much CS experience did you have before you became an intern?</b><br/>
+                Not much - I took AP CS in high school and didn’t learn a ton,
+                and when I applied to be an intern I was taking CS 31.
+                <br/><br/><b>Are you still an officer in ACM? If so, in what Committee?</b><br/>
+                Yes! I’m the director of ACM Impact!
+                <br/><br/><b>What advice would you give to prospective interns?</b><br/>
+                Try to maximize the amount of rejections you get - try everything!
```

Comment:
Instead of using `<br/>` and `<b>`, let's (at the very least) use `<p>` and header tags.

Reply:
I don't want the `<p>` formatting, so I think I'm going to use just headers instead
or I could do `<p>` and add a class to modify the formatting if you think there should be a `<p>`

Reply:
If you want different formatting, I would just override the style - tags are fundamentally about the semantic meaning of the content, not the styling itself.